<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeueBuchung</class>
 <widget class="QWidget" name="NeueBuchung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>735</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 118, 114);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Header">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>1241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>Neue Buchung</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonZurueck">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: #ffffff;
    font: 600 italic 15pt &quot;Segoe UI&quot;;
    padding: 10px 20px;
    border: 2px solid #2980b9;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1f618d;
}
</string>
   </property>
   <property name="text">
    <string>Zurück</string>
   </property>
  </widget>
  <widget class="QLabel" name="betragText">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Betrag:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kategorieText">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Kategorie:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="datumText">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Datum:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="regelmaessigText">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>regelmässig ?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="notesText">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Notes:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBestaetigen">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>610</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

        background-color: rgb(40, 53, 132);
    color: #ffffff;
    font: 600 italic 15pt &quot;Segoe UI&quot;;
    padding: 10px 20px;
    border: 2px solid #2980b9;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #2c82c9;
}

QPushButton:pressed {
    background-color: #1869ab;
}
</string>
   </property>
   <property name="text">
    <string>Bestätigen</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="betrag">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>210</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 italic 15pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:15pt; font-weight:600; font-style:italic;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="notes">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>530</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 italic 15pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:15pt; font-weight:600; font-style:italic;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonEinnahme">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton {
    padding: 5px;
    border: 2px solid #555;
    border-radius: 8px;
    background-color: #f2f2f2;
    color: #333;
}


QRadioButton:checked {
    background-color: #66bb6a;
    color: white;
}


QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border-radius: 7px;
    background-color: #ccc;
}


QRadioButton::indicator:checked {
    background-color: #66bb6a;
}
</string>
   </property>
   <property name="text">
    <string>Einnahme</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupTyp_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonAusgabe">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton {
    padding: 5px;
    border: 2px solid #555;
    border-radius: 8px;
    background-color: #f2f2f2;
    color: #333;
}


QRadioButton:checked {
    background-color: #66bb6a;
    color: white;
}


QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border-radius: 7px;
    background-color: #ccc;
}


QRadioButton::indicator:checked {
    background-color: #66bb6a;
}
</string>
   </property>
   <property name="text">
    <string>Ausgabe</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupTyp_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonRechnung">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    padding: 5px;
    border: 2px solid #555;
    border-radius: 8px;
    background-color: #f2f2f2;
    color: #333;
}


QRadioButton:checked {
    background-color: #66bb6a;
    color: white;
}


QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border-radius: 7px;
    background-color: #ccc;
}

QRadioButton::indicator:checked {
    background-color: #66bb6a;
}
</string>
   </property>
   <property name="text">
    <string>Rechnung</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupTyp_2</string>
   </attribute>
  </widget>
  <widget class="QDateEdit" name="dateEditDatum">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>280</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {

        font: 600 italic 15pt &quot;Segoe UI&quot;;
        color: rgb(0, 0, 0);
    padding: 5px;
    background-color: #f5f5f5;
    selection-background-color: #66bb6a;
}

QDateEdit::drop-down {
    width: 20px;
    image: url(':/icons/calendar_icon.png');
    subcontrol-position:
    subcontrol-origin: padding;
    right: 5px;
}

QDateEdit::down-arrow {
    image: none;
}

QDateEdit::down-arrow:hover {
    background-color: #ddd;
}</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="faelligkeitsdatumText">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Fälligkeitssdatum
 (für Rechnungen)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditFaelligkeitsdatum">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {

        font: 600 italic 15pt &quot;Segoe UI&quot;;
        color: rgb(0, 0, 0);
    padding: 5px;
    background-color: #f5f5f5;
    selection-background-color: #66bb6a;
}

QDateEdit::drop-down {
    width: 20px;
    image: url(':/icons/calendar_icon.png');
    subcontrol-position:
    subcontrol-origin: padding;
    right: 5px;
}

QDateEdit::down-arrow {
    image: none;
}

QDateEdit::down-arrow:hover {
    background-color: #ddd;
}</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonJa">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton {
    padding: 5px;
    border: 2px solid #555;
    border-radius: 8px;
    background-color: #f2f2f2;
    color: #333;
}


QRadioButton:checked {
    background-color: #66bb6a;
    color: white;
}


QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border-radius: 7px;
    background-color: #ccc;
}


QRadioButton::indicator:checked {
    background-color: #66bb6a;
}
</string>
   </property>
   <property name="text">
    <string>Ja</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupRegelmaessig</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonNein">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>360</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    padding: 5px;
    border: 2px solid #555;
    border-radius: 8px;
    background-color: #f2f2f2;
    color: #333;
}


QRadioButton:checked {
    background-color: #66bb6a;
    color: white;
}


QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border-radius: 7px;
    background-color: #ccc;
}


QRadioButton::indicator:checked {
    background-color: #66bb6a;
}
</string>
   </property>
   <property name="text">
    <string>Nein</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupRegelmaessig</string>
   </attribute>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
        font: 600 italic 15pt &quot;Segoe UI&quot;;
        color: rgb(0, 0, 0);
    padding: 5px;
    background-color: #f5f5f5;
    selection-background-color: #66bb6a;
    border: 1px solid #707070;
    border-radius: 3px;
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
        font: 600 italic 15pt &quot;Segoe UI&quot;;
        color: rgb(0, 0, 0);
    padding: 5px;
    background-color: #f5f5f5;
    selection-background-color: #66bb6a;
    border: 1px solid #707070;
    border-radius: 3px;
}
</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButtonZurueck</zorder>
  <zorder>betragText</zorder>
  <zorder>kategorieText</zorder>
  <zorder>datumText</zorder>
  <zorder>regelmaessigText</zorder>
  <zorder>notesText</zorder>
  <zorder>Header</zorder>
  <zorder>pushButtonBestaetigen</zorder>
  <zorder>betrag</zorder>
  <zorder>notes</zorder>
  <zorder>radioButtonEinnahme</zorder>
  <zorder>radioButtonAusgabe</zorder>
  <zorder>radioButtonRechnung</zorder>
  <zorder>dateEditDatum</zorder>
  <zorder>faelligkeitsdatumText</zorder>
  <zorder>dateEditFaelligkeitsdatum</zorder>
  <zorder>radioButtonJa</zorder>
  <zorder>radioButtonNein</zorder>
  <zorder>comboBox</zorder>
  <zorder>comboBox_2</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupRegelmaessig"/>
  <buttongroup name="buttonGroupTyp_2"/>
 </buttongroups>
</ui>
